### DISCUSSION TITLE
[Request] Progress Indicator

### DISCUSSION THREAD

**Discussion by jacmag** [microsoft/finnts#109] [Ideas]
Would it be possible to add a progress indicator while the model is running? The text output is very helpful, but it doesn't indicate how many time series are complete and how many are remaining. Progress is even more unclear when using parallel processing. Thank you.

**Comment by mitokic** [comment]
Having a progress bar when parallel_processing is set to local_machine is hard to do. I've researched potential solutions but none seem to work properly when using %dopar% in combination with foreach to run things in parallel. If you used the azure_batch or spark option, you should get an indicator on how many time series are complete and how many are still running in Azure.
